ngraph/frontend/paddlepaddle/src/op/range.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "range.hpp"
+#include <ngraph/opsets/opset6.hpp>
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            namespace op
+            {
+                NamedOutputs range(const NodeContext& node)
+                {
+                    auto start = node.get_ng_input("Start");
+                    auto stop = node.get_ng_input("End");
+                    auto step = node.get_ng_input("Step");
+                    auto type = node.get_out_port_types("Out");
```

Comment:
Please check that there is exactly 1 element in `type` or all element types are equal. Or use API provided especially for this case: `node.get_out_port_type("Out")`

Reply:
Change to `node.get_out_port_type("Out")` fix in b1970c6c81b09a8ae0833ff4a0af8471019e166b